packages/app/src/components/Auth/AuthProvider.tsx

```diff
@@ -4,17 +4,18 @@
  *
  * This code is for demonstration purposes and should not be considered production ready.
  *--------------------------------------------------------------------------------------------*/
-import { AccessToken, UserInfo } from "@bentley/itwin-client";
```

Comment:
i think the `type AccessToken` still exists, its just in core-common.

curious did you run the codemod, or manually go through updating everything?

Reply:
Ran the codemod, which didnt take care of this at all. And the codemod repo metions that UserInfo have been moved to appui-react, but it's been removed since. 

I wanted to remove all the client, but I needed the `UserInfo` for some LD and AI tasks, so I kept it from 2.19. And given that `type AccessToken = string;` sound missleading to me, I used `string` instead. (because itwin-client AccessToken != core-common AccessToken...)

Reply:
fair point re the access token type

guess need to update the code-mod docs